code/modules/economy/account.dm

```diff
@@ -49,11 +50,15 @@
 				bank_card_talk("ERROR: Payday aborted, departmental funds insufficient.")
 				return FALSE
 			else
-				bank_card_talk("Payday processed, account now holds $[account_balance].")
+				bank_card_talk("Payday processed, account now holds $[account_balance].")		
 				return TRUE
 	bank_card_talk("ERROR: Payday aborted, unable to contact departmental account.")
 	return FALSE
 
+
+/datum/bank_account/proc/add_neetbux()
```

Comment:
I know you're very committed to this joke, but perhaps you could just call this proc "add_welfare"

Reply:
👀 thats what it says